README.md

```diff
@@ -71,4 +71,7 @@ To use one of the themes, just include the theme's CSS file in your page. Exampl
 [![a11y Dark](screenshots/prism-a11y-dark.png)](themes/prism-a11y-dark.css)
 
 * [__Dracula__](themes/prism-dracula.css) (by [Byverdu](https://github.com/byverdu))<br />
-[![a11y Dark](screenshots/prism-dracula.png)](themes/prism-dracula.css)
\ No newline at end of file
+[![a11y Dark](screenshots/prism-dracula.png)](themes/prism-dracula.css)
+
+* [__Synthwave '84__](themes/prism-synthwave84.css) (originally by [Robb Owen](https://github.com/robb0wen), adapted by [Marc Backes](https://github.com/themarcba))<br />
+[![CB](screenshots/prism-synthwave84.png)](themes/prism-synthwave84.css)
```

Comment:
```suggestion
[![Synthwave '84](screenshots/prism-synthwave84.png)](themes/prism-synthwave84.css)
```

Reply:
The screenshot is also bigger than the others. 250px x 180px seems to be the average.

Reply:
@ahmadawais You're not wrong but I'm not going to block this PR over it.